docs/source/asr/configs.rst

```diff
@@ -114,6 +114,41 @@ You can use any combination of Cutout, frequency/time SpecAugment, or none of th
 
 See the `Audio Augmentors <./api.html#Audio Augmentors>`__ API section for more details.
 
+Tokenizer Configurations
+------------------------
+
+Some models utilize sub-word encoding via an external tokenizer instead of explicitly defining their vocabulary.
+
+For such models, a ``tokenizer`` section is added  to the model config. ASR Models currently support two types of
+custom tokenizers.
+
```

Comment:
Explain the current supported tokenizers? Maybe some suggestions on which one to use? or even what sizes?

Reply:
mentioning that tokenizers is needed for which models exactly? and what to set them for char-based models?